security/openconnect/src/opnsense/mvc/app/models/OPNsense/Openconnect/General.xml

```diff
@@ -37,14 +37,43 @@
                     <sha1>SHA1</sha1>
                 </OptionValues>
         </hash>
-        <group type="TextField">
+        <compression type="OptionField">
+            <multiple>N</multiple>
+            <Required>N</Required>
+                <OptionValues>
+                    <none>No Compression</none>
+                    <stateless>Stateless</stateless>
+                    <all>Stateful</all>
+                </OptionValues>
+        </compression>
+        <authgroup type="TextField">
             <Required>N</Required>
             <mask>/^[a-zA-Z0-9]{1,64}$/</mask>
             <ValidationMessage>Only a-zA-Z0-9 are allowed.</ValidationMessage>
-        </group>
+        </authgroup>
+        <usergroup type="TextField">
+            <Required>N</Required>
+            <mask>/^[a-zA-Z0-9]{1,64}$/</mask>
+            <ValidationMessage>Only a-zA-Z0-9 are allowed.</ValidationMessage>
+        </usergroup>
         <clientcertificate type="CertificateField">
             <Type>cert</Type>
             <Required>N</Required>
         </clientcertificate>
+        <passtos type="BooleanField">
+            <default>1</default>
+            <Required>N</Required>
+        </passtos>
+        <pfs type="BooleanField">
+            <default>0</default>
+            <Required>N</Required>
+        </pfs>
+        <reconnecttime type="AutoNumberField">
+            <MinimumValue>1</MinimumValue>
+            <MaximumValue>900</MaximumValue>
+            <default>300</default>
+            <Required>N</Required>
```

Comment:
if it is not required, default is not really needed but you can keep it as it is.